components/StoryModal.tsx

```diff
@@ -35,11 +36,13 @@ const Section = ({ title, children }): JSX.Element => (
 interface EditableFields {
   description?: string;
   owners?: Owner[];
+  reviews: Review[];
 }
 
 const getEditableFields = (story: Story): EditableFields => ({
   description: story?.description,
   owners: story?.owners || [],
+  reviews: story?.reviews || [],
```

Comment:
nice! you may need to do a mapping to review_ids like I did for owners and labels in the save, but i'm glad to see this pattern being used!